q2_fmt/tests/test_engraftment.py

```diff
@@ -487,3 +488,364 @@ def test_alpha_dists_with_extra_samples_in_diversity_not_in_metadata(self):
 
     def test_examples(self):
         self.execute_examples()
+
+
+class TestPeds(TestBase):
+    def test_check_get_donor(self):
```

Comment:
```suggestion
    def test_get_donor(self):
```
I would remove the 'check' portion of this test name since (from my understanding) it just means that you are confirming the correct donor is being pulled from your method.